packages/amplication-data-service-generator/src/server/static/src/providers/secrets/secretsManager.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+import { SecretsManagerServiceBase } from "./base/secretsManager.service.base";
+
```

Comment:
add a comment here to explain to the user to replace the default behavior with a link to docs